tools/ci_connector_ops/ci_connector_ops/qa_engine/cloud_availability_updater.py

```diff
@@ -1,47 +1,58 @@
 #
-# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
```

Comment:
I think it's 2023...

Reply:
The autoformatting is still in 2022 :(

Reply:
https://github.com/airbytehq/airbyte/pull/22576 updated it :)